packages/core/src/epics/websocket-kernel.js

```
@@ -55,3 +64,34 @@ export const launchWebSocketKernelEpic = (action$: *, store: *) =>
       );
     })
   );
+
+export const interruptKernelEpic = (action$: *, store: *) =>
+  action$.pipe(
+    ofType(INTERRUPT_KERNEL),
+    filter(action => {
```

I like the idea of just ignoring these, but then again, that's sort-of a silent fail, isn't it?

---

ignoring which part here, the action?

---

yah, the action.